config/config_test.go

```diff
@@ -17,17 +16,14 @@ func TestParseTerragruntConfigRemoteStateMinimalConfig(t *testing.T) {
 	t.Parallel()
 
 	config := `
-terragrunt = {
-  remote_state {
-    backend = "s3"
-  }
+remote_state {
+	backend = "s3"
+	config = {}
```

Comment:
Fixed: https://github.com/gruntwork-io/terragrunt/pull/731/commits/249662ef03cb27ea3a06a3a688da1493f20b901f. Shouldn't matter at all now that we're using the real HCL parser instead of regex.